org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -401,4 +419,14 @@ private void verifyApiClient(String connectionId) {
               .build();
     }
   }
+
+  /**
+   * Checks if this container has been marked unhealthy and if so, forces a shutdown via
+   * System.exit().
+   */
+  private void checkContainerHealth(ShutdownTrigger trigger) {
```

Comment:
I got a little confused when I read this. The function name made me think it will be returning information about the container's health, but it's instead conditionally shutting down the container. To fix this, I'd recommend updating the name to something more action-oriented, such as `shutdownIfUnhealthy` or `conditionallyShutdownContainer`.

Reply:
Good call - I'll rename to `shutdownContainerIfUnhealthy`